tests/dummy/app/app.js

```diff
@@ -0,0 +1,18 @@
+import Ember from 'ember'
+import Resolver from 'ember/resolver'
+import loadInitializers from 'ember-load-initializers'
+import config from './config/environment'
+
+var App
+
+Ember.MODEL_FACTORY_INJECTIONS = true
+
+App = Ember.Application.extend({
```

Comment:
Not really clear on why the `Ember.MODEL_FACTORY_INJECTIONS = true` has to come before the `var App` and `App = ...`. Couldn't we just have:
```js
Ember.MODEL_FACTORY_INJECTIONS = true
const App = Ember.Application.extend({
```
??